app/models/ability.rb

```diff
@@ -127,6 +127,7 @@ def signed_in_with_organizer_role(user)
     can :manage, Vday, conference_id: conf_ids_for_organizer
     can :manage, CallForPaper, conference_id: conf_ids_for_organizer
     can :manage, Event, conference_id: conf_ids_for_organizer
+    can :schedule, Event, conference_id: conf_ids_for_organizer
```

Comment:
It doesn't. Either you explicitly allow only specific methods and remove `:manage` or you use `:manage`. Everything else is just misleading IMHO